test/Exchanger.ts

```diff
@@ -15,22 +30,52 @@ describe("Exchanger Test suit", function () {
         // Setup & deploy contracts
         r = await buildContractTestContext();
         lvUSD = r.lvUSD;
+        ousd = r.externalOUSD;
         owner = r.owner;
+        exchanger = r.exchanger;
+        coordinator = r.coordinator;
 
-        // Mint 200 ETH of LvUSD for owner
-        await lvUSD.mint(owner.address, ethers.utils.parseUnits("200.0"));
-        // Swap 200 ETH of 3CRV for owner
-        await helperSwapETHWith3CRV(owner, ethers.utils.parseUnits("200.0"));
+        // Fund exchanger
+        await lvUSD.mint(exchanger.address, amountStarting);
+        await helperSwapETHWithOUSD(owner, amountStarting);
+        await ousd.transfer(exchanger.address, amountStarting);
     });
 
     describe("Exchanges", function () {
-        // TODO
-        it("Should swap LvUSD for OUSD", async function () {
-            expect(true);
+        it("Tests should init with some funds", async function () {
+            expect(await lvUSD.balanceOf(exchanger.address)).eq(amountStarting);
+            expect(await ousd.balanceOf(exchanger.address)).eq(amountStarting);
+        });
+
+        // TODO test a revert of imbalanced pool
+
+        // Create position
+        describe("swapLvUSDforOUSD()", function () {
+            beforeEach(async function () {
+                await exchanger.swapLvUSDforOUSD(amountToExchange);
```

Comment:
Having this call in  beforeEach and noit resetting network means it will run twice before OUSD test (once for lvUSD test and than another time before OUSD test)